sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClient.java

```diff
@@ -144,6 +144,14 @@ public synchronized void start() {
             return;
         }
         logger.info("Starting a new event processor instance.");
+        try {
+            partitionBasedLoadBalancer.checkBlobStatus().block();
```

Comment:
This block should have a timeout interval... `.block()` by default will be indefinite. `.block(retryOptions.getOperationTimeout)`?